bugsnag-android-core/src/androidTest/java/com/bugsnag/android/BreadcrumbsSerializationTest.kt

```diff
@@ -34,22 +29,6 @@ class BreadcrumbsSerializationTest {
         client?.close()
     }
 
-    /**
-     * Verifies that breadcrumb metadata is serialised
-     */
-    @Test
-    fun testPayloadType() {
-        val metadata = mapOf(Pair("direction", "left"))
-        breadcrumbs.add(Breadcrumb("Rotated Menu", BreadcrumbType.STATE, metadata))
-
-        val json = streamableToJsonArray(breadcrumbs)
-        val node = json.getJSONObject(0)
-        assertEquals("Rotated Menu", node.get("name"))
```

Comment:
Is this breadcrumb metadata now tested elsewhere?

Reply:
Updated the test case so that metadata is now included in the expected JSON.